recipes/libcudss/test/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.12)
+project(all_headers VERSION 0.1 LANGUAGES C CXX CUDA)
+
+find_package(cudss)
+
+set(ALL_TARGETS
+    cudss
```

Comment:
@kvoronin, I added a compiler test, but it fails because `libcudss_static.a` is missing (as per channel policy we don't ship static libraries). This needs to be fixed upstream. cudss should allow cmake to find the cudss package without finding the static library (the static library should be an optional component).